sql/values_test.go

```diff
@@ -18,74 +18,60 @@
 package sql
 
 import (
+	"fmt"
 	"reflect"
-	"strconv"
 	"testing"
 
-	"github.com/cockroachdb/cockroach/sql/driver"
 	"github.com/cockroachdb/cockroach/sql/parser"
 )
 
-func TestProcessSelect(t *testing.T) {
-	s := (*Server)(nil) // enough for now
+func TestValues(t *testing.T) {
+	p := &planner{}
```

Comment:
No. Though the `planner` methods take a pointer so escape analysis will convert this to a heap allocated object. This is test code, though, so it doesn't really matter.